source/credentials_provider_imds.c

```diff
@@ -201,9 +204,17 @@ static int s_credentials_provider_imds_get_credentials_async(
         goto error;
     }
 
+    AWS_LOGF_INFO(
```

Comment:
Need to dig farther, but I don't think this is an accurate statement.  Credentials have not been successfully retrieved, they've only successfully started the retrieval process.

Reply:
Yep, you're right. Fixed (and added a couple more about each step of the process).